src/sentry/db/models/fields/bounded.py

```diff
@@ -71,9 +71,7 @@ class BoundedBigAutoField(models.AutoField):
 
         def db_type(self, connection):
             engine = connection.settings_dict['ENGINE']
-            if 'mysql' in engine:
-                return "bigint AUTO_INCREMENT"
-            elif 'oracle' in engine:
+            if 'oracle' in engine:
```

Comment:
Can we uhh, remove the oracle references too? I assume those came in from copy/paste.

Reply:
Lol, sure

Reply:
Removed a couple references to mssql as well, amazing